saleor/graphql/shipping/types.py

```diff
@@ -76,3 +82,8 @@ def resolve_countries(root: models.ShippingZone, *_args):
     @staticmethod
     def resolve_shipping_methods(root: models.ShippingZone, *_args):
         return root.shipping_methods.all()
+
+    @staticmethod
+    @permission_required(ProductPermissions.MANAGE_PRODUCTS)
+    def resolve_warehouses(root: models.ShippingZone, *_args):
```

Comment:
I almost all ower simple resolvers we use arguments `(root: <TYPE>, _info):` instead of `(root: <TYPE>, *_args):` can you make resolvers in this file more coherent to other.